dask/base.py

```diff
@@ -850,9 +850,10 @@ def tokenize(*args, **kwargs):
     >>> tokenize('Hello') == tokenize('Hello')
     True
     """
+    token = str(tuple(map(normalize_token, args)))
     if kwargs:
-        args = args + (kwargs,)
-    return md5(str(tuple(map(normalize_token, args))).encode()).hexdigest()
+        token += str(tuple(map(normalize_token, (kwargs,))))
+    return md5(token.encode()).hexdigest()
```

Comment:
GitHub won't let me suggest since there's deleted lines, but a more efficient fix would be:

```python
hasher = md5(str(tuple(map(normalize_token, args))).encode())
if kwargs:
    hasher.update(str(normalize_token(kwargs)).encode())
return hasher.hexdigest()

Reply:
Thanks for the suggestion -- agreed that looks better 👍